.gitlab/corona-jobs.yml

```diff
@@ -27,11 +27,22 @@ corona_clang_hip:
 # ------------------------------------------------------------------------------
 # HIP + TPLs
 # ------------------------------------------------------------------------------
-corona_clang_hip_tpls:
+corona_rocmcc_550_tpls:
   parallel:
     matrix:
-      - COMPILER_SPEC: rocmcc@5.4.1
+      - COMPILER_SPEC: rocmcc@5.5.0
         AMDGPU_TARGET: [gfx906]
+        ROCM_VERSION: 5.5.0
   variables:
-    SPEC: "%${COMPILER_SPEC} cstd=99 cxxstd=14 precision=double ~int64 amdgpu_target=${AMDGPU_TARGET} scheduler=flux +rocm+mpi+magma+raja ^magma+rocm amdgpu_target=${AMDGPU_TARGET} ^raja+rocm~openmp~examples~exercises amdgpu_target=${AMDGPU_TARGET}"
+    SPEC: "%${COMPILER_SPEC} cstd=99 cxxstd=14 precision=double ~int64 amdgpu_target=${AMDGPU_TARGET} scheduler=flux +rocm+mpi+magma+raja+kokkos+kokkos-kernels~ginkgo ^magma+rocm amdgpu_target=${AMDGPU_TARGET} ^raja+rocm~openmp~examples~exercises amdgpu_target=${AMDGPU_TARGET} ^kokkos+rocm~profiling amdgpu_target=${AMDGPU_TARGET} ^hipblas@${ROCM_VERSION} ^hipsparse@${ROCM_VERSION} ^hip@${ROCM_VERSION} ^hsa-rocr-dev@${ROCM_VERSION} ^llvm-amdgpu@${ROCM_VERSION}"
+  extends: .corona_build_and_test
+
+corona_rocmcc_523_tpls:
```

Comment:
Looks like this test is failing 
```
CMake Error at cmake/tpl/SundialsGinkgo.cmake:43 (find_package):
  Could not find a package configuration file provided by "Ginkgo" with any
  of the following names:
    GinkgoConfig.cmake
    ginkgo-config.cmake
  Add the installation prefix of "Ginkgo" to CMAKE_PREFIX_PATH or set
  "Ginkgo_DIR" to a directory containing one of the above files.  If "Ginkgo"
-- Configuring incomplete, errors occurred!
  provides a separate development package or SDK, be sure it has been
See also "/dev/shm/g/g14/dgardner/.jacamar-ci/builds/DcCeyZKx/000/gitlab/sundials/sundials/build_1328297_/CMakeFiles/CMakeOutput.log".
  installed.
See also "/dev/shm/g/g14/dgardner/.jacamar-ci/builds/DcCeyZKx/000/gitlab/sundials/sundials/build_1328297_/CMakeFiles/CMakeError.log".
Call Stack (most recent call first):
  cmake/SundialsSetupTPLs.cmake:67 (include)
  CMakeLists.txt:170 (include)
2023-07-18T20:57:15.597775Z broker.err[0]: rc2.0: .gitlab/build_and_test.sh Exited (rc=1) 170.8s
flux-job: task(s) exited with exit code 1
```

Reply:
This was a file system permissions issue that I now fixed.